RNFastImage.podspec

```diff
@@ -16,6 +16,6 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/**/*.{h,m}"
 
   s.dependency 'React-Core'
-  s.dependency 'SDWebImage', '~> 5.11.1'
+  s.dependency 'SDWebImage', '~> 5.18.7'
```

Comment:
I would suggest to use more relaxed version requirements like:
```
s.dependency 'SDWebImage', '~> 5.18'
s.dependency 'SDWebImageWebPCoder', '~> 0.14'
```
This is more conventional in podspecs unless we have a really good reason to restrict it.